identity/resource_token.go

```diff
@@ -98,13 +104,11 @@ func ResourceToken() *schema.Resource {
 			Type:     schema.TypeInt,
 			Optional: true,
 			ForceNew: true,
-			Default:  0,
 		},
 		"comment": {
 			Type:     schema.TypeString,
 			Optional: true,
 			ForceNew: true,
-			Default:  "",
```

Comment:
Maybe just make it "Terraform" by default and have no if/else in the code?...

Reply:
As discussed above, empty comments should be allowed